src/Workspaces/CSharp/Portable/Extensions/SyntaxNodeExtensions_SharedWithCodeStyle.cs

```diff
@@ -219,5 +219,31 @@ public static bool IsEmbeddedStatementOwner([NotNullWhen(returnValue: true)] thi
                 WhileStatementSyntax n => n.Statement,
                 _ => null,
             };
+
```

Comment:
The main problem with these small PRs might be they can now end up causing merge conflicts in the refactoring PRs as this file has been moved into shared project. I am not sure though.